Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -881,29 +883,81 @@ def template_test(template: str):
                         f' The available Templates for this instance: {", ".join(template_names)}.')
 
 
-@logger
+@polling_function(
+    name=demisto.command(),
+    interval=60,
+    timeout=1200
+)
+def pan_os_command_polling(args: dict, result):
+    """
+    Polls job status until it is done
+    """
```

Comment:
is this function a general polling for the `pan-os` command?

Reply:
Yes. The use case here is polling a tech support file export to keep checking until it is done. This currently has to be done using the `pan-os` command (i.e. the command below); it does not have its own command. We could add a command for it, but there are so many commands in this integration already I thought it might be best to add polling to the generic `pan-os` command.

`!pan-os type=export category=tech-support action=status job-id=<JOB_ID>`